src/services/auth.service.ts

```diff
@@ -0,0 +1,50 @@
+import { instance } from '../api/axios.api';
+import {
+  IResetPasswordRequest,
+  IUserRegistration,
+  IResponseUserData,
+  IResetPassword,
+  IUserData,
+  IUserId,
+  IUser,
+} from '../types/types';
+
+export const AuthService = {
+  async registration(
+    userData: IUserRegistration,
+  ): Promise<IResponseUserData | undefined> {
```

Comment:
why do we return undefined in each method?

Reply:
undefined here will be returned only if an error happened or the data was not received